### PR TITLE
Bug 1782740: Addressing LF causing cert rotation

### PR DESCRIPTION
**Pull request by ewolinetz**
Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1782740

Able to recreate the master-certs secret and elasticsearch secret being recreated by deploying CLO, creating a log forwarding object, then deleting the CLO pod. No longer observe that with this PR.

### CODE REVIEW

**Comment by ewolinetz** [comment]
issue seems to still be present

File: pkg/controller/forwarding/forwarding_controller.go [hunk 1]
```diff
@@ -89,40 +89,54 @@ func (r *ReconcileForwarding) Reconcile(request reconcile.Request) (reconcile.Re
 			// Return and don't requeue
 			return reconcile.Result{}, nil
 		}
-		// Error reading the object - requeue the request.
-		return reconcile.Result{}, err
+		// Error reading the object -- requeue
+		return reconcileResult, err
```
Comment (richm) [inline pkg/controller/forwarding/forwarding_controller.go]: So when you want the request to be requeued, you return `reconcileResult`, and when you do not want the result to be requeued, you return `reconcile.Result{}`?

Comment (ewolinetz) [reply pkg/controller/forwarding/forwarding_controller.go]: yes, `reconcileResult` has a field `RequeueAfter` that puts this back on the event queue

**Comment by ewolinetz** [comment]
Still want to do more regression testing before removing the `[WIP]`

**Pull request merged**